sql/sql_zip_dict.h

```diff
@@ -57,6 +57,13 @@ inline bool is_hardcoded(const dd::String_type &schema,
                                 table_name == "compression_dictionary_cols"));
 }
 
+/** @return true if the table is hardcoded compression dictionary tables
```

Comment:
"is a hardcoded compression dictionary table"
or
"is one of hardcoded compression dictionary tables"